src/features/home/DashboardPanelRight.tsx

```diff
@@ -1,23 +1,64 @@
 import './dashboardPanelRight.less';
 import Icon from '@ant-design/icons';
 import { Button, Col, Row, Typography } from 'antd';
+import Big from 'big.js';
+import { useMemo } from 'react';
 import { useNavigate } from 'react-router-dom';
 
+import type { RootState } from '../../app/store';
+import { useTypedSelector } from '../../app/store';
 import { ReactComponent as depositIcon } from '../../assets/images/deposit-green.svg';
 import { FEE_DEPOSIT_ROUTE, FEE_WITHDRAW_ROUTE } from '../../routes/constants';
 import type { LbtcUnit } from '../../utils';
-import { fromSatsToUnitOrFractional } from '../../utils';
+import { fromSatsToUnitOrFractional, LBTC_COINGECKOID } from '../../utils';
 import { useGetFeeBalanceQuery } from '../operator/operator.api';
+import type { PriceFeedQueryResult } from '../rates.api';
 
 const { Title } = Typography;
 
 interface DashboardPanelRightProps {
   lbtcUnit: LbtcUnit;
+  priceFeed: PriceFeedQueryResult;
 }
 
-export const DashboardPanelRight = ({ lbtcUnit }: DashboardPanelRightProps): JSX.Element => {
+export const DashboardPanelRight = ({ lbtcUnit, priceFeed }: DashboardPanelRightProps): JSX.Element => {
   const navigate = useNavigate();
+  const { currency } = useTypedSelector(({ settings }: RootState) => settings);
   const { data: feeBalance } = useGetFeeBalanceQuery();
+  const { data: prices, isLoading: isLoadingPrices, isError: isErrorPrices } = priceFeed;
+
+  const FeeAccountBalance = useMemo(() => {
```

Comment:
Use the helper function `convertAssetAmountToCurrency` to compute `fiatBalance`. Don't use useMemo